### DISCUSSION TITLE
Increase blog content width and size of overall text & Categories page issue

### DISCUSSION THREAD

**Discussion by pranavdhawale** [thuliteio/doks#1252] [Q&A] [answered]
Hey!
I want my blog width to be as shown in the image below. And I also want to increase the overall text size of the text used for blog and blog list

![image](https://github.com/gethyas/doks/assets/74247818/0de0953f-01f9-40b9-b1d5-33adc24a4055)

**ALSO**

The categories page doesn't display the categories it just shows the arrow and not the name of the category

![image](https://github.com/gethyas/doks/assets/74247818/b66ef695-0b1a-4860-8887-37a8bb0a8425)

**Comment by h-enk** [comment, marked as answer]
- Copy `node_modules/@hyas/doks-core/layouts/blog/single.html` and paste as `layouts/blog/single.html`. Update the latter (line 15) with something like:

    ```html
    <div class="col-md-13">
    ```
- Add to `assets/scss/common/_custom.scss` something like:
    ```scss
    .single.blog .content,
    .list.blog .content {
      font-size: $font-size-base * 1.375;
    }
    ```

- That's a bug — thx! Will be fixed. Quick fix for now: copy `node_modules/@hyas/doks-core/layouts/_default/terms.html` and paste as `layouts/_default/terms.html`.  Update the latter (line 12) with:
```html
              <a class="stretched-link" href="{{ .RelPermalink }}">{{ .Title }} &rarr;</a>
```

**Reply by pranavdhawale** [reply]
> That's a bug — thx!

Yeah! I was facing this issue with many themes, but didn't thought it as a bug

**Reply by pranavdhawale** [reply]
> Add to assets/scss/common/_custom.scss something like:

this is not working, no changes occuring

**Reply by h-enk** [reply]
Works for me with a fresh install

![2024-06-18_09-10-25](https://github.com/gethyas/doks/assets/3902872/1b990de6-97bc-4ddf-afff-231741336875)

Did you clear the Hugo build directories and files before running the development server, for example:

```bash
npx shx rm -rf public resources .hugo_build.lock && npm run dev
```

**Reply by pranavdhawale** [reply]
> Did you clear the Hugo build directories and files before running the development server, for example:

Yep.

With custom css:
![image](https://github.com/gethyas/doks/assets/74247818/df66ce03-66e0-40ec-a674-74286da2bc7f)

Without custom css:
![image](https://github.com/gethyas/doks/assets/74247818/f3603e2a-6a3e-4351-a0ae-3d37318ef3e5)

If you are saying it widens the content, but no i did by setting `col-xl-10`
I want to increase the overall font size of the blog

**Reply by h-enk** [reply]
Have a look at how I did it:

- https://github.com/h-enk/doks-1252/blob/main/assets/scss/common/_custom.scss
- https://github.com/h-enk/doks-1252/blob/main/layouts/blog/single.html#L25
- https://github.com/h-enk/doks-1252/blob/main/layouts/_default/terms.html#L12

Results:

- https://lambent-unicorn-546d39.netlify.app/blog/example-post/
- https://lambent-unicorn-546d39.netlify.app/categories/

**Reply by pranavdhawale** [reply]
can you check my code?? My font size is remaining the same
Repo: https://github.com/ByteBlogs/byteblogs

**Reply by h-enk** [reply]
It's because of this definition: https://github.com/ByteBlogs/byteblogs/blob/main/assets/scss/common/_custom.scss#L24

Add this code: 

```scss
.single.blog .content,
.list.blog .content {
  font-size: $font-size-base * 1.375;
}
```

And make it more specific (or remove the referenced line above)

**Reply by h-enk** [reply]
So:

```scss
.single.blog .content p,
.list.blog .content p {
  font-size: $font-size-base * 1.375;
}
```
 Probably also add `ol`, `ul` and so on

**Reply by pranavdhawale** [reply]
Unfortunately, its not working for me. can you try it out on my code, please?

**Reply by h-enk** [reply]
Check the repo commits to see what I've done:

- https://github.com/h-enk/byteblogs
- https://aesthetic-stardust-50557f.netlify.app/

**Reply by pranavdhawale** [reply]
Thanks!...
